develop/develop-images/build_enhancements.md

```diff
@@ -189,3 +189,39 @@ Once the `Dockerfile` is created, use the `--ssh` option for connectivity with t
 ```bash
 $ docker build --ssh default .
 ```
+
+## Troubleshooting : issues with private registries
+
+#### x509: certificate signed by unknown authority
+If you are fetching images from insecure registry (with self-signed certificates) and/or using such aa registry as a mirror, you are facing a known issue in Docker 18.09 :
```

Comment:
`s/aa/a`

Also alternatively the user can set [insecure-registries](https://docs.docker.com/registry/insecure/)